internal/access/passwordreset.go

```diff
@@ -5,20 +5,19 @@ import (
 
 	"github.com/gin-gonic/gin"
 
-	"github.com/infrahq/infra/internal/logging"
 	"github.com/infrahq/infra/internal/server/data"
 	"github.com/infrahq/infra/internal/server/models"
 )
 
-func PasswordResetRequest(c *gin.Context, email string, ttl time.Duration) (token string, user *models.Identity, err error) {
+func PasswordResetRequest(c *gin.Context, email string, ttl time.Duration) (string, *models.Identity, error) {
```

Comment:
unnamed string return arguments are less clear than named ones.

Reply:
Ya, especially true when there are multiple return values with the same type. Usually when they have different types it's pretty clear.

It's unfortunate that naming them also extends the scope of the variable (to any place in the function, instead of only after declaration). I can add these back